feature/repair/src/main/java/com/iteneum/repair/data/RepairViewModel.kt

```diff
@@ -0,0 +1,83 @@
+package com.iteneum.repair.data
+
+import android.net.Uri
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.iteneum.RepairModel
+import com.iteneum.network.DataState
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * File that contains [RepairViewModel] that works as ViewModel for the Repair View.
+ * It will handle the data work & process when loading & be done with the view.
+ *
+ * @author Jose Miguel Garcia Reyes
+ */
+
+
+@HiltViewModel
+class RepairViewModel @Inject constructor(): ViewModel() {
+    private val _dataInfo : MutableState<RepairModel?> = mutableStateOf(null)
```

Comment:
For the moment it will be like that.
Based in Sergio & Carlos comments, he instructed to add the Unit to the RepairView, since it has to show the Department/House number. Because of that, it will be mostly probable that the View will have too type of perspectives. For the Employer, to be able to manipulate the field; and for the employee, to see the value already inserted. In case that it cannot be attached or found, we have to enable the null possibility.
What I will add for now is a TODO to verify this later on.

Reply:
Agree. I will make the change

Reply:
I made the change that instead of creating it null, I will make an initialization of the RepairModel data class. Is this better?

New declaration
![image](https://github.com/irv205/droid_leaspert/assets/25538206/2835ed97-35d0-4c84-b4d3-5a5b37c113fe)

Reply:
Change made but pending for new comments